cns/middlewares/SWIFTv2.go

```diff
@@ -0,0 +1,103 @@
+package middlewares
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/Azure/azure-container-networking/cns"
+	"github.com/Azure/azure-container-networking/cns/configuration"
+	"github.com/Azure/azure-container-networking/cns/types"
+	"github.com/Azure/azure-container-networking/crd/multitenancy/api/v1alpha1"
+	v1 "k8s.io/api/core/v1"
+	k8stypes "k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var ErrMTPNCNotReady = errors.New("mtpnc is not ready")
+
+type SWIFTv2Middleware struct {
+	cli client.Client
```

Comment:
export this and drop the constructor, the constructor isn't doing anything but assigning this. We only use constructors in Go when we need to set some default values or run some logic